### PR TITLE
Fix torch `TypedStorage` issue. Add throughput as a metric in training summary.

### PR DESCRIPTION
**Pull request by loctxmoreh**
Two little changes:
1. Fix [this issue](https://github.com/moreh-dev/ml-workbench/issues/122) about `torch.storage.TypedStorage` on our `torch==1.13.1`
2. Measuring throughput and add them to training report.
  
I thought the throughput measuring would be complicated, since the training script takes into account distributed training with multiple GPUs. But since we have no multi GPU setup to run on, I go with a naive solution.

### CODE REVIEW

**Comment by loctxmoreh** [comment]
I cannot add @huonglarne and @HDThang as reviewers, somehow :shrug:

**Comment by yeomni** [comment]
@loctxmoreh Thanks for the heads up! Trying to put them on the list of available reviewers.

**Review by yeomni — APPROVED** [review]
LGTM

**Review by kimmorehvn — APPROVED** [review]
FYI
